### DISCUSSION TITLE
Cannot configure Rockson WF96L

### DISCUSSION THREAD

**Discussion by dakkar78** [make-all/tuya-local#1725] [Q&A] [answered]
Hi!

I just installed a Rockson WF96L water level controller.
It works well and was immediately recognised with the Tuya Smart App, with no issues.
I got the Device ID and local key on the Tuya IoT Platform and am now trying to get it configured with Tuya Local.
I can see there is already a file for this device in the custom_component folder - `konlen_wf96l_waterlevel_controller.yaml` (Konlen/Rockson is essentially the same device), but when I try to add the device, I get an error message saying "Unable to connect to your device with those details."

I tried with the specific IP address, but I got the same error...

Not sure what I am doing wrong and what I'm supposed to do? Any suggestion?

Thanks!

**Comment by make-all** [comment, marked as answer]
There are many possibilities discussed in #522
As well as sometimes needing to explicitly set the IP address, you also sometimes need to set the protocol version, especially on newer 3.4 and 3.5 devices.

**Reply by dakkar78** [reply]
Thanks! It worked with IP address and protocol version 3.4.
